compute_endpoint/tests/integration/endpoint/executors/high_throughput/test_htex_regression.py

```diff
@@ -0,0 +1,68 @@
+import queue
+import random
+import uuid
+
+import pytest
+from globus_compute_common import messagepack
+from globus_compute_endpoint.engines import HighThroughputEngine
+from globus_compute_sdk.serialize import ComputeSerializer
+from tests.utils import double, ez_pack_function
+
+
+@pytest.fixture
+def engine():
+    ep_id = uuid.uuid4()
+    engine = HighThroughputEngine(
+        label="HTEXEngine",
+        heartbeat_period=1,
+        worker_debug=True,
+    )
+    results_queue = queue.Queue()
+    engine.start(endpoint_id=ep_id, run_dir="/tmp", results_passthrough=results_queue)
+
+    yield engine
+    engine.shutdown()
```

Comment:
A friendly suggestion to consider the `tmp_path` fixture over the raw string of `"/tmp"`.  Consider:

```python
@pytest.fixture
def engine(tmp_path):
    ...
    engine.start(..., run_dir=tmp_path, ...)
    ...
```

If it _truly_ needs a string, then can stringify it for the engine:
```python
    engine.start(..., run_dir=str(tmp_path), ...)
```